### ISSUE TITLE
add link to list of allowed max versions close to addon upload link

### ISSUE THREAD

**Issue by opto** [thundernest/addons-server#182]
### Describe the problem and steps to reproduce it:
(Please include as many details as possible.)

my local TB installs addon xpi with forbidden max versions.

So for clarity, it would be helpful to have the link to the allowed max versions close to the upload button.

That would help to avoid unnecessary rejected runs through the linter. And even if the linter tells that the max version is not accepted .. where do i find the accepted values if that link is not visible.

### Anything else we should know?
(Please include a link to the page, screenshots and any relevant files.)

**Comment by RealRaven2000** [comment]
I am pretty sure this information (list of accepted min / max versions) is here:

https://addons.thunderbird.net/en-US/thunderbird/pages/appversions/

**Comment by opto** [comment]
yes, my proposal is to include that link next to the addon upload button, so that it has higher visibility

**Comment by jobisoft** [comment]
We also added it to the list of important resources here:
https://developer.thunderbird.net/add-ons/resources
